server/routes/index.js

```diff
@@ -30,6 +31,23 @@ module.exports = (
 ) => {
   const router = express.Router();
 
+  router.use(
+    [
+      '^/$',
+      '/content',
+      '/npr',
+      '/tags',
+      '/topics',
+      '/timetable',
+      '/money',
+      '/approved-visitors',
+      '/profile',
+      '/games',
+      '^/search/?$',
+    ],
+    retrieveTopicList(cmsService),
```

Comment:
@jonhindle  you're list vs ours: 

```
'/', '/topics', '/timetable', '/money', '/approved-visitors', '/profile', '/content', '/npr', '/tags', '/games']
```

think its just search that differs? we're ensuring it doesn't fire for the suggestion api, maybe that would suffice for you?